src/framework/Elsa.Studio.Core/Elsa.Studio.Core.csproj

```diff
@@ -25,12 +25,12 @@
         <PackageReference Include="ThrottleDebounce"/>
     </ItemGroup>
 
-<!--    <ItemGroup Label="Elsa">-->
-<!--        <ProjectReference Include="..\..\..\..\..\elsa-core\main\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>-->
-<!--    </ItemGroup>-->
-
     <ItemGroup Label="Elsa">
-        <PackageReference Include="Elsa.Api.Client"/>
+        <ProjectReference Include="..\..\..\..\..\elsa-core\main\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>
```

Comment:
Will fix this once the PR in elsa-core has been merged and built into new pakages.